plataform_front_end/public/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+    <link href='https://fonts.googleapis.com/css?family= Baloo Da' rel='stylesheet' type='text/css'>
```

Comment:
I couldn't find this font on google fonts. I think this is not the right URL. I only found the [Baloo Da 2](https://fonts.google.com/specimen/Baloo+Da+2), it this the font you want?

If so, you have to import as Google suggests:
```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Baloo+Da+2:wght@400;500;600;700;800&display=swap" rel="stylesheet">
```

You will have to change the `font-family` on [this line](https://github.com/maclararose/Homer-Project/pull/1/files#diff-0fe06ade15eb90b929da8946870ab97c4abaead3364f71467081cf6521bdca3cR20) as well, to apply the right font-family as Google suggests to:
```css
    font-family: 'Baloo Da 2', cursive;
 ```
 
![Screenshot from 2021-06-10 20-34-51](https://user-images.githubusercontent.com/54550926/121610048-83e29080-ca2b-11eb-9011-aa3cc8fbaae5.png)